calicoctl/commands/resources.go

```diff
@@ -124,6 +124,16 @@ func getResourceFromArguments(args map[string]interface{}) (unversioned.Resource
 			p.Metadata.CIDR = *cidr
 		}
 		return *p, nil
+	case "bgpPeer":
```

Comment:
A couple quick questions:

1) As it is now I think we're pretty strict about wording.  kubectl lets you do things like:
```
kubectl get networkpolicy
kubectl get networkpolicies
```
both of which return the same stuff, which is pretty nice in terms of UX.  Definitely _not_ something to fix in this PR, but might be worth an issue in the GA milestone.